crates/context_aware_config/src/api/config/helpers.rs

```diff
@@ -1,28 +1,6 @@
-use super::types::{Config, Context};
-use actix_web::web::Query;
-use serde_json::{json, Map, Value};
-use std::collections::HashMap;
-use superposition_macros::{bad_argument, unexpected_error};
-use superposition_types::{result as superposition, Cac, Overrides};
-
-pub fn filter_context(
-    contexts: &[Context],
-    dimension_data: &Map<String, Value>,
-) -> Vec<Context> {
-    contexts
-        .iter()
-        .filter_map(|context| {
-            match jsonlogic::partial_apply(
-                &json!(context.condition),
-                &json!(dimension_data),
-            ) {
-                Ok(jsonlogic::PartialApplyOutcome::Resolved(Value::Bool(true)))
-                | Ok(jsonlogic::PartialApplyOutcome::Ambiguous) => Some(context.clone()),
-                _ => None,
-            }
-        })
-        .collect()
-}
```

Comment:
INFO: Moved to [here](https://github.com/juspay/superposition/pull/264/files#diff-7109af13c8c50c7956803d8af88403472e464bd9e291d7bfd3374f4eda3aba00R9-R26)